### PR TITLE
Add instructions for enabling natural language search

### PR DESCRIPTION
**Pull request by munkyshi**
### Linked issue(s):
Fixes: KOL-3501

### What change does this PR introduce and why?
Provide documentation example for extracting image search embeddings

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [x] Relevant docs have been added / updated

### CODE REVIEW

File: docs/advanced-usage/enabling-natural-language-search.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+icon: kolena/comparison-16
+---
+
+# :kolena-comparison-20: Enabling Natural Language Search
```
Comment (munkyshi) [inline docs/advanced-usage/enabling-natural-language-search.md]: Not sure if `enabling` is the right verb here, as this wouldn't be a one time toggle, but I also wanted this to feel more approachable. Thoughts?

File: docs/advanced-usage/enabling-natural-language-search.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    pip install ./kolena_embeddings-*.*.*.tar.gz
+    ```
+
+=== "`poetry`"
+
+    ```shell
+    poetry add ./kolena_embeddings-*.*.*.tar.gz
+    ```
+
+This package provides the `kclip.util.extract_and_upload_embeddings` method:
```
Comment (munkyshi) [inline docs/advanced-usage/enabling-natural-language-search.md]: Just realized we still have this as `kclip` -- thoughts on if this is an issue?

File: docs/advanced-usage/enabling-natural-language-search.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    "s3://kolena-public-datasets/labeled-faces-in-the-wild/imgs/Aaron_Eckhart/Aaron_Eckhart_0001.jpg",
+]
+extract_and_upload_embeddings(iter_image_locators(locators))
+```
+
+Once the upload completes, we can now visit [<nobr>:kolena-studio-16: Studio</nobr>](https://app.kolena.io/redirect/studio)
+to search by natural language over the corresponding [`Image`][kolena.workflow.Image] data.
+
+## Conclusion
+In this tutorial, we learned how to extract and upload vector embeddings over your [`Image`][kolena.workflow.Image] data.
```
Comment (munkyshi) [inline docs/advanced-usage/enabling-natural-language-search.md]: We don't actually have a guide on how to use NLS once the embeddings are uploaded. Does it seem like we need to add one? That feels like a whole different beast to document...

File: docs/advanced-usage/enabling-natural-language-search.md (hunk 1 above)
Comment (y27choi) [reply docs/advanced-usage/enabling-natural-language-search.md]: How about `How to Set Up Natural Language Search` or `Natural Language Search Setup`?

File: docs/advanced-usage/enabling-natural-language-search.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    Please message your point of contact for the latest relevant extractor package.
+
+## Example
+
+The `kolena/kolenaIO` contains a runnable [example](https://github.com/kolenaIO/kolena/tree/trunk/examples/search_embeddings)
+of integration code for embeddings extraction and upload.
+This builds off the data uploaded in the [age_estimation](https://github.com/kolenaIO/kolena/tree/trunk/examples/age_estimation)
+example workflow, and is best run after this data has been uploaded to your Kolena platform.
+
+## How to Enabling Natural Language and Search on Kolena
```
Comment (y27choi) [inline docs/advanced-usage/enabling-natural-language-search.md]: Should this be `How to Enable Natural Language Search on Kolena`?

File: docs/advanced-usage/enabling-natural-language-search.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+extract_and_upload_embeddings(iter_image_locators(locators))
+```
+
+Once the upload completes, we can now visit [<nobr>:kolena-studio-16: Studio</nobr>](https://app.kolena.io/redirect/studio)
+to search by natural language over the corresponding [`Image`][kolena.workflow.Image] data.
+
+## Conclusion
+In this tutorial, we learned how to extract and upload vector embeddings over your [`Image`][kolena.workflow.Image] data.
+
+## FAQ
```
Comment (y27choi) [inline docs/advanced-usage/enabling-natural-language-search.md]: Woo I like the FAQ section. I think one of the common questions we get is whether they need to upload the embeddings for each test suite (even when it's the same set of images).

File: docs/advanced-usage/enabling-natural-language-search.md (hunk 2 above)
Comment (munkyshi) [reply docs/advanced-usage/enabling-natural-language-search.md]: Renamed to kembeddings -- also putting together associated PR from the corresponding `kolena embeddings` repo

**Review by y27choi — APPROVED** [review]
The doc looks good.  I tried it out with the latest change in the embeddings extraction repo and it worked as expected 👍

**Pull request merged**